### DISCUSSION TITLE
【求助】绳网能不能按照热度排序帖子捏

### DISCUSSION THREAD

**Discussion by fancy-fantasy** [share121/inter-knot#2954] [其他]
![三月七](https://github.com/user-attachments/assets/bdd562bb-769e-4536-9dcc-e85b77e98237)
绳网的生物多样性有点离谱了<img src="https://img-static.mihoyo.com/communityweb/upload/460a345beb85c449f354a38e2cd8e01c.png" alt="佩拉 懵" width="75" >
想看看有无糕手绳匠知道怎么做

**Comment by fancy-fantasy** [comment]
快三点了还在发帖我也是绳斤<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >

**Comment by I-ikun** [comment]
我也觉得<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
